sales.html

```diff
@@ -7,13 +7,30 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
     </head>
     <body>
-        <h1>Salmon Cookies: Hourly Sales Projections</h1>
-        <table>
-            <caption>Cookies Needed By Location Each Day</caption>
-            <thead></thead>
-            <tbody></tbody>
-            <tfoot></tfoot>
-        </table>
+        <article>
+            <h1>Salmon Cookies: Hourly Sales Projections</h1>
+            <table>
+                <caption>Cookies Needed By Location Each Day</caption>
+                <thead></thead>
+                <tbody></tbody>
+                <tfoot></tfoot>
+            </table>
+        </article>
+        <article>
+            <h2>New Store Input</h2>
+            <form>
+                <label for="store-name">Store Name</label>
+                <input type="text" id="store-name">
+                <p id="hide-name-warning-message">Please enter a valid store name.</p>
```

Comment:
As mentioned in the CSS, perhaps we could give this an Id of something like `error-message`, then toggle its display using a `hide` class instead of toggling its `Id`.